### PR TITLE
add permission to mount qlaunch romfs

### PR DESCRIPTION
**Pull request by HookedBehemoth**
Waiting for: https://github.com/switchbrew/switch-tools/pull/27

This is needed to have access to the applet sounds from application override.
fsOpenDataFileSystemByProgramId only allows mounting registered titles so I chose qlaunch as it runs all the time.

### CODE REVIEW

File: hbl.json [hunk 1]
```diff
@@ -161,4 +166,4 @@
             }
         }
     ]
-}
+}
```
Comment (fincs) [inline hbl.json:169]: Revert removal of newline at eof, please

**Comment by fincs** [comment]
This will be merged once switch-tools package is updated.

**Comment by HookedBehemoth** [comment]
I've just now seen that a switch-tools release was drafted but not built.
Was that intentional?

**Comment by fincs** [comment]
No it is not. We're kind of distracted dealing with other packages atm, a bit of further patience would be most appreciated.

**Pull request merged**